src/pages/homepage/RisingSection/RisingItem.tsx

```diff
@@ -0,0 +1,55 @@
+/** @jsxImportSource @emotion/react */
+import styled from "@emotion/styled";
+import type { Product } from "@/types/api_types";
+
+export default function RisingItem({ product }: { product: Product }) {
+  const { name, imageURL, price, brandInfo } = product;
```

Comment:
```suggestion
export default function RisingItem({ name, imageURL, price, brandInfo }: Product) {
```
이렇게 product가 props가 되도록 해주면 어떨까요?
그리고 이렇게 작성해주면 렌더링을 할 때 어떤 차이가 있을까요!?

Reply:
더 세밀한 리렌더링 트리거링을 할 수 있을것으로 생각됩니다!